packages/flutter/lib/src/services/text_input.dart

```diff
@@ -2226,7 +2226,13 @@ class _PlatformTextInputControl with TextInputControl {
   Map<String, dynamic> _configurationToJson(TextInputConfiguration configuration) {
     final Map<String, dynamic> json = configuration.toJson();
     if (TextInput._instance._currentControl != _PlatformTextInputControl.instance) {
-      json['inputType'] = TextInputType.none.toJson();
+      final Map<String, dynamic> none = TextInputType.none.toJson();
+      /// See: https://github.com/flutter/flutter/issues/125875
+      /// On Web engine, use forceMultiline to create <input> or <textarea> element
+      /// when customizing [TextInputControl].
+      /// We send forceMultiline to the engine only when customizing [TextInputControl].
```

Comment:
nit: if this is the only flag that web engine uses to determine whether to create a textarea or an input, maybe call it `isMultiline` instead of "force"?

Reply:
Also I'm not sure what "We send forceMultiline to the engine only when customizing [TextInputControl]" means here. Do you mean it's only needed when we set TextInputType to none?

Reply:
@LongCatIsLooong 
1. Have replaced `forceMultiline` to `isMultiline`.
2. Sorry! I am not good at English. It means it's only needed when the user registers a custom `TextInputControl`. Is it clear now? If it is not clear, please write it for me here and I will copy it.